frame/support/src/lib.rs

```diff
@@ -1237,7 +1237,7 @@ pub mod pallet_prelude {
 		traits::{Get, Hooks, IsType, GetPalletVersion, EnsureOrigin, PalletInfoAccess},
 		dispatch::{DispatchResultWithPostInfo, Parameter, DispatchError, DispatchResult},
 		weights::{DispatchClass, Pays, Weight},
-		storage::types::{StorageValue, StorageMap, StorageDoubleMap, ValueQuery, OptionQuery},
+		storage::types::{StorageValue, StorageMap, StorageDoubleMap, StorageNMap, ValueQuery, OptionQuery},
```

Comment:
maybe we can import `Key` too so that people can write:
```
	#[pallet::storage]
	#[pallet::getter(fn nmap)]
	pub type NMap<T> = StorageNMap<_, Key<Blake2_128Concat, u8>, u32>;
```

Reply:
I thought of that, but `Key` is a common enough name for a struct, and importing it here in the pallet prelude would cause breaking changes to pallets that already have a `Key` struct defined.

Reply:
Good point, but Maybe we can import it as `NMapKey` or `StorageKey` or similar to disambiguate.

Reply:
well `StorageKey ` is used all over the place in substrate already, so please not that :P